daisy_workflows/image_import/inspection/tests/linux_tests.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
```

Comment:
Who runs these tests? It looks like these are E2E tests. Consider renaming the folder to indicate this.

Reply:
Currently nothing is executing them. Any ideas of where they could be invoked?

I wouldn't characterize these as E2E tests, as they provide a fake filesystem, rather than mounting and inspecting a real disk. The design doc talks about tests that use VM images -- those are more aligned with the E2E tests that you're thinking of, since they would execute all of the code here.

Reply:
Ideally, we would have another tab in our e2e test dashboard.